lib/command/ls.php

```diff
@@ -28,6 +29,8 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+require_once __DIR__ . '/../../vendor/autoload.php';
+
 class ls extends Command {
```

Comment:
cammelcase?

Reply:
I need to rework namespacing and class names in a future PR